test/local/db.js

```diff
@@ -234,9 +218,16 @@ describe('redis enabled', () => {
     const DB = proxyquire(`${LIB_DIR}/db`, {
       './pool': function () { return pool },
       redis: { createClient: () => redis }
-    })({ tokenLifetimes, redis: {enabled: false} }, log, tokens, {})
+    })({ tokenLifetimes, redis: {enabled: true} }, log, tokens, {})
```

Comment:
Turns out these `redis enabled` tests actually had redis disabled, lol. Fixed and added some extra tests that fail when redis is disabled.